src/gleam_fetch_ffi.mjs

```diff
@@ -81,3 +89,37 @@ export async function read_json_body(response) {
     return new Error(new InvalidJsonBody());
   }
 }
+
+export function newFormData() {
+  return new FormData()
+}
+
+function cloneFormData(formData) {
```

Comment:
How come this is implemented in JS using mutation rather than being a pure data structure in Gleam?

Reply:
Because in my mind, the idea was to use the `FormData` object provided by the runtime, and not a `FormData` created by us. For interoperability purpose, you could for example construct the `FormData` on the JS side, or on the Gleam side, and use it in the other world, without other work, because it would just work.